docs/includes/openapi/openapi-ui.adoc

```diff
@@ -0,0 +1,200 @@
+///////////////////////////////////////////////////////////////////////////////
+
+    Copyright (c) 2022 Oracle and/or its affiliates.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+///////////////////////////////////////////////////////////////////////////////
+
+ifndef::rootdir[:rootdir: {docdir}/../..]
+// Following make editing the included file a little easier
+:flavor-lc: se
+:flavor-uc: SE
+:se-flavor:
+
+// tag::preamble[]
+:feature-name: Helidon OpenAPI U/I support
+:screen-capture-start: openapi-ui-screen-capture-greeting-{flavor-lc}-start.png
+:screen-capture-expanded: openapi-ui-screen-capture-greeting-{flavor-lc}-expanded.png
+// end::preamble[]
+
+// tag::intro[]
+
+== Contents
+
+- <<Overview, Overview>>
+- <<Maven Coordinates, Maven Coordinates>>
+- <<Usage, Usage>>
+- <<API, API>>
+- <<Configuration, Configuration>>
+- <<Additional Information, Additional Information>>
+
+// end::intro[]
+
+// tag::overview[]
+== Overview
+
+SmallRye offers an link:{smallrye-openapi-ui-base-url}[OpenAPI user interface component] which displays a web page based on your application's OpenAPI document.
+Through that U/I, users can invoke the operations declared in the document.
+While not generally suitable for end-users, the OpenAPI U/I can be useful for demonstrating and "test driving" your service's endpoints.
```

Comment:
OpenAPI UI  --- this should be a global change to the docs. UI is the preferred abbreviation of user interface (not U/I).

Reply:
Done.